libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SFSDKWebViewStateManager.m

```diff
@@ -82,25 +83,35 @@ + (void)removeUIWebViewCookies:(NSArray *)cookieNames fromDomains:(NSArray *)dom
 
 + (void)removeWKWebViewCookies:(NSArray *)domainNames withCompletion:(nullable void(^)(void))completionBlock {
     NSAssert(domainNames != nil && [domainNames count] > 0, ERR_NO_DOMAIN_NAMES);
-    WKWebsiteDataStore *dateStore = [WKWebsiteDataStore defaultDataStore];
+    WKWebsiteDataStore *dataStore = [WKWebsiteDataStore defaultDataStore];
     NSSet *websiteDataTypes = [NSSet setWithArray:@[ WKWebsiteDataTypeCookies]];
-    [dateStore fetchDataRecordsOfTypes:websiteDataTypes
+    [dataStore fetchDataRecordsOfTypes:websiteDataTypes
                      completionHandler:^(NSArray<WKWebsiteDataRecord *> *records) {
                          NSMutableArray<WKWebsiteDataRecord *> *deletedRecords = [NSMutableArray new];
-                         for ( WKWebsiteDataRecord * record in records) {
+                         for (WKWebsiteDataRecord * record in records) {
+                             // Cookie record display names look like "salesforce.com", "force.com". Make
```

Comment:
As near as I can tell, all cookie domains get coalesced down to the last two parts of the domain name, and one `WKWebsiteDataRecord` represents all cookies in that bucket.  More granular cookie management APIs come in iOS 11, but before that, this will have to suffice.